python/tvm/relay/frontend/onnx.py

```diff
@@ -530,10 +530,7 @@ class Mod(OnnxOpConverter):
     @classmethod
     def _impl_v1(cls, inputs, attr, params):
         assert len(inputs) == 2, "Mod op take 2 inputs, {} given".format(len(inputs))
-        if attr['fmod'] == 1:
-            op_name = "floor_mod"
-        else:
-            op_name = "mod"
+        op_name = "mod"
```

Comment:
Some handling for `fmod=1` will need to be added here. From the onnx operator definition, `fmod=1` implies that the sign of the dividend should be kept in the output. When the dividend is negative, this differs from default relay behavior. We'll probably have to multiply the output by `relay.sign(inputs[0])` when `fmod=1` to make this work.